src/webgpu/api/operation/onSubmittedWorkDone.spec.ts

```diff
@@ -0,0 +1,40 @@
+export const description = `
+Tests for the behavior of GPUQueue.onSubmittedWorkDone().
+
+Note that any promise timeouts will be detected by the framework.
+`;
+
+import { makeTestGroup } from '../../../common/framework/test_group.js';
+import { range } from '../../../common/framework/util/util.js';
+import { GPUTest } from '../../gpu_test.js';
+
+export const g = makeTestGroup(GPUTest);
+
+g.test('without_work')
```

Comment:
We should also check the ordering of onSubmittedWorkDone with respect to each other, and with respect to mapAsync callbacks.

Reply:
Latter is tracked here https://github.com/gpuweb/cts/blob/main/src/webgpu/api/operation/async_ordering/README.txt